repose-aggregator/core/repose-core/src/test/java/org/openrepose/nodeservice/request/RequestHeaderServiceContextTest.java

```diff
@@ -55,8 +52,6 @@
 import static org.mockito.Matchers.eq;
 import static org.mockito.Mockito.*;
 
-@RunWith(PowerMockRunner.class)
-@PowerMockIgnore("javax.management.*")
```

Comment:
Why do most of our tests written in Scala have:
```
@RunWith(classOf[JUnitRunner])
```
but a few don't have any at all?

Is this being brought in from somewhere else?
Why the inconsistency?